setup.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+from pip.req import parse_requirements
+import setuptools
+
+# Filters out relative/local requirements (i.e. ../lib/utils)
+remote_requirements = '\n'.join(str(r.req) for r in parse_requirements("requirements.txt", session='dummy') if r.req)
+
+setuptools.setup(
+    name='kafka-streams-examples-tests',
+    version='0.0.1',
+
+    author="Confluent, Inc.",
+    author_email="partner-support@confluent.io",
```

Comment:
this seems like it's not the right email address for this repo, shouldn't this be to either the streams team or some more general address? i don't think there is anything partner related in this repo

Reply:
`pint` should be able to bump the POM and setup.py in one go.